requirements/dev.txt

```diff
@@ -1,5 +1,5 @@
 -r base.txt
-flake8
+flake8==3.5.0
 pre-commit
```

Comment:
Can you pin these other versions too, with what you have in your system?

Reply:
Sure.